README.md

```diff
@@ -6,67 +6,63 @@ Using fluent-plugin-logdna, you can send the logs you collect with Fluentd to Lo
 
 ## Instructions
 
-* Install [Fluentd](http://www.fluentd.org/download)
-* Alternative install if using fluentd package manager (td-agent): `td-agent-gem install fluent-plugin-logdna`
-* Add the contents below to `/etc/fluent/fluent.conf`. For td-agent, use `/etc/td-agent/td-agent.conf`:
-* Alternative install without td-agent is: `gem install fluent-plugin-logdna`
+* Install Fluentd
+  * [Download here](http://www.fluentd.org/download)
+  * If using fluentd package manager (td-agent): `td-agent-gem install fluent-plugin-logdna`
+  * To install without td-agent: `gem install fluent-plugin-logdna`
+* Add the config below to `/etc/fluent/fluent.conf`. For td-agent, `/etc/td-agent/td-agent.conf`:
+
+## Configuration
+
+### Configuration Parameters
+- `api_key`: [Ingestion Key](https://docs.logdna.com/docs/ingestion-key), *Required*
+- `hostname`: Hostname, *Required*
+- `app`: App Name, *Optional*
+- `mac`: MAC Address, *Optional*
+- `ip`: IP Address, *Optional*
```

Comment:
All these optional ones need to be in brackets.  That's industry standard to denote optional.

Reply:
@darinspivey, like
- `ip`: IP Address *(Optional)* ?

Can u share the link explaining the industry standard since @alfeng6 asked for that too? Thanks!

Reply:
No, like saying "optional" and "required" is superfluous.  It's denoted by brackets, so:
```
[ip]: <String> IP address
requiredThing: <String> Something required
```

Sorry, can't give you a link.  This is a decades-old style.

Reply:
For example, the node client states `createLogger(key[, options])` meaning all the `options` are optional :]

Reply:
I see - again, comparing with the code library doesn't work since there is no such functionality here - it's just configure and let it stream - no need for types either and whoever doesn't know whether `[]` brackets mean being `Optional` can easily get confused as well, I think

Reply:
Sure, but it's language independent :]. I'm just illustrating how you do "optional" things.  We should do that here, IMO

Reply:
Done, @darinspivey!

Reply:
There are varying views on this, I suppose.  Feel free to leave it.

Reply:
They should definitely have types tho

Reply:
Actually, I'm used to that syntax for years in command lines. Gotta say, describing config files--not so much. Never really in this format, though. Usually, an option list, indicating what's required, what isn't... Perhaps we're trying to fit a square peg into a round hole here? (or vice versa...)